CMakeLists.txt

```diff
@@ -300,7 +301,19 @@ install(DIRECTORY "${NVFUSER_SRCS_DIR}/"
   DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/nvfuser"
   FILES_MATCHING
   PATTERN "*.h"
-  PATTERN "csrc/C++20/type_traits")
+  PATTERN "csrc/C++20/type_traits"
+  PATTERN "csrc/struct.inl")
+
+# TODO guard including flatbuffers headers
+# installing flatbuffers headers
+install(DIRECTORY "${NVFUSER_THIRD_PARTY_DIR}/flatbuffers/include/flatbuffers/"
+  DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/nvfuser/flatbuffers")
+
+# installing dynamic_type headers
+install(DIRECTORY "${NVFUSER_SRCS_DIR}/lib/dynamic_type/src/"
+  DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/nvfuser/"
```

Comment:
any reason that we don't have a separate dynamic_type folder for the header?

Reply:
I moved the `dynamic_type` headers to a separate folder.